src/System.Private.ServiceModel/tests/Common/Scenarios/ScenarioTestTypes.cs

```diff
@@ -1332,4 +1332,20 @@ public string Content
         get { return _content; }
         set { _content = value; }
     }
+}
+
+[DataContract]
```

Comment:
Can you please set Name and Namespace of the DataContract as we did for other types? Without them, the test could be easily broken if we move the test types around, e.g. put it under another namespace.